examples/README.md

```diff
@@ -87,6 +87,7 @@ Example | File | Description
 `mesh2d` | [`2d/mesh2d.rs`](./2d/mesh2d.rs) | Renders a 2d mesh
 `mesh2d_manual` | [`2d/mesh2d_manual.rs`](./2d/mesh2d_manual.rs) | Renders a custom mesh "manually" with "mid-level" renderer apis.
 `rect` | [`2d/rect.rs`](./2d/rect.rs) | Renders a rectangle
+`circle` | [`2d/circle.rs`](./2d/circle.rs) | Renders a circle
```

Comment:
IMO this and `rect` should be merged into a larger `shapes` example.

Reply:
agreed!